tests/test_events.py

```diff
@@ -41,11 +41,15 @@ def get_trainer(self, precision='fp32', **kwargs):
                 batch_size=train_batch_size,
                 sampler=dist.get_sampler(train_dataset),
             ),
-            eval_dataloader=DataLoader(
+            eval_dataloader=(DataLoader(
                 dataset=eval_dataset,
                 batch_size=8,
                 sampler=dist.get_sampler(eval_dataset),
-            ),
+            ), DataLoader(
+                dataset=eval_dataset,
+                batch_size=4,
+                sampler=dist.get_sampler(eval_dataset),
+            )),
```

Comment:
minor nit: can you split these out into a separate line? it's starting to look a little nasty. Really lint should be complaining about this...

Reply:
I changed the formatting to what you asked, and the linter turned it back into this.

Reply:
lol

How about something like: 
```
evaluator1 = ...
evaluator2 = ...
```

Reply:
implemented as above